src/main/java/de/focusshift/zeiterfassung/usermanagement/WorkingTimeController.java

```diff
@@ -95,6 +100,8 @@ String newWorkingTime(@PathVariable("userId") Long userId, Model model,
                           @AuthenticationPrincipal OidcUser principal) {
 
         final WorkingTimeDto workingTimeDto = new WorkingTimeDto();
+        workingTimeDto.setFederalState(FederalState.NONE);
+        workingTimeDto.setWorksOnPublicHoliday(false);
```

Comment:
das finde ich super und könnte man so in die UV übernehmen, sodass sie nur angezeigt werden.